### ISSUE TITLE
Add global table search

### ISSUE THREAD

**Issue by dosumis** [brain-bican/taxonomy-development-tools#61]

**Comment by dosumis** [comment]
@hkir-dev - could you estimate how much work involved given above spec.  (Suspect it might not be so easy, but I think worth it)

**Comment by jeremymiller** [comment]
Agreed that search is critical.  Current search is great, but global search would be better, if it is doable.
